CHANGES.md

```diff
@@ -1,6 +1,6 @@
 wavesurfer.js changelog
 =======================
-5.3.0 (unreleased)
+6.0.0 (unreleased)
 ------------------
 - add additional type to `waveColor` and `progressColor` parameters to support linear gradients (#2345)
 - Regions plugin: increase region z-index to fix stacking inconsistencies (#2353)
```

Comment:
the changelog entry for this PR is missing

Reply:
added